Framework/advanced.json

```diff
@@ -76,14 +64,7 @@
       "id": "tst2",
       "active": "true",
       "machines": [],
-      "dataHeaders": [
-        {
-          "binding": "data",
-          "dataOrigin": "TST",
-          "dataDescription": "DATA"
-        }
-      ],
-      "subSpec": "*",
+      "query" : "data:TST/RAWDATA",
```

Comment:
I am wondering if it would not be better to be explicit here : 
```
"query" : "data:TST/RAWDATA/*",
```
What do you think ?

Reply:
Or maybe it is imposed by upstream ?

Reply:
In fact the version with * is not properly parsed and I forgot to check out if this is a bug or a lack of feature. @ktf Does the query syntax expect * for subSpec, like in `"data:TST/RAWDATA/*"`?

Reply:
I would need to check but I am on holidays. Maybe the syntax is simply “data:TST/RAWDATA”?

Reply:
Or: “data:TST/RAWDATA/$1”

Reply:
I maintain that it would be better to be explicit here. However, we can proceed like that.

Reply:
So neither of `data:TST/RAWDATA/*` and `data:TST/RAWDATA/$1` work, the only way now is to leave it unspecified.

Reply:
You are right, I will try to add it.